README.md

```diff
@@ -23,17 +23,17 @@ In particular it demonstrates `Nexus Plugin`.
   ![Docker Registry Nexus Bearer token](images/docker-bearer-token.png "Docker Bearer Token")
   - Write down the Docker Registry URL:
   ```
-  export DOCKER_REGISTRY=`oc get route docker-registry -n nexus -o jsonpath='{.spec.host}{"\n"}'`
+  export DOCKER_REGISTRY=`oc get route docker-repository -n nexus -o jsonpath='{.spec.host}{"\n"}'`
```

Comment:
I would vote for keeping the "docker-registry" string here and instead, renaming the Route.

What I am exposing is indeed a docker registry, but I mixed it up here. So correct term is "docker-registry" and Route name should follow that convention.

Reply:
No  problem, but can you please fix the route name so it matches with the instruction? No rush. Thank you!